lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.html

```diff
@@ -21,9 +27,14 @@
             (matChipInputTokenEnd)="add($event)">
     </mat-chip-list>
     <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selected($event)" id="adf-search-chip-autocomplete">
-        <mat-option [disabled]="option | adfIsIncluded: selectedOptions" *ngFor="let option of filteredOptions$ | async"
-                    [ngClass]="(option | adfIsIncluded: selectedOptions) && 'adf-autocomplete-added-option'">
-            {{option}}
-        </mat-option>
+        <ng-container *ngIf="optionInput.value.length > 0">
+            <mat-option
+                [disabled]="option.id ? (option.id | adfIsIncluded: selectedOptions: 'id') : (option.value | adfIsIncluded: selectedOptions: 'value')"
```

Comment:
Should not this be configurable somehow? What if somebody would like to use something different than id field? For example 'name' field?